docs/mobile-apps/features/fast-booted-simulators.md

```diff
@@ -0,0 +1,107 @@
+
+Sauce Labs supports the following configurations: 
+
+- iPhone 14 Simulator with the latest iOS 16.x version (current_major).
+- iPhone 13 Simulator with the latest iOS 15.x version (previous_major).
+- iPad (10th generation) Simulator with the latest iOS 16.x version (current_major).
+- iPad (9th generation) Simulator with the latest iOS 15.x version (previous_major).
+
+:::note
+By using the latest/current_major version, you can test your apps in the most up-to-date environment, while the previous_major version provides the environment for testing apps on the previous iOS version.
+:::
+
+These Simulator configurations can be used separately for automated tests, allowing you to target specific devices and iOS versions for testing. This flexibility can help you optimize the testing process and improve the compatibility and performance of your apps.
+
+## What You'll Need
+
+- A Sauce Labs account ([Log in](https://accounts.saucelabs.com/am/XUI/#login/) or sign up for a [free trial license](https://saucelabs.com/sign-up)).
+- A native iOS, or iPadOS mobile app.
+
+## Accessing Fast Booted iOS Simulator for Live Testing
+
+To access the Fast Booted iOS Simulator follow the steps below, for this example we will use the iPhone 14 with the latest iOS 16.x version (current_major):
+
+1. On Sauce Labs click **Live** > **Cross Browser** > **Mobile Virtual**.
+<img src={useBaseUrl('img/mobile-apps/fast-sim-1.png')} alt="fast booted menu" width="750"/>
+
+2. Select “Apple” as the Manufacturer.
+3. If you want to use the “current_major” version, then use these values :
+- Select **iPhone 14 Simulator** for the Device.
+- Select the latest available version of iOS.
+
+
+:::note
+If you want to use the “previous_major” version, then use these values:
+- Select “iPhone 13 Simulator” for the Device
+- Select “iOS 15.4” for the OS Version
+:::
+
+## Accessing Fast Booted iOS Simulator for Automated Testing with Appium
+
+To use the fast-booted Simulators, specify the following device configuration capabilities:
+
+For iPhone:
+
+```java
+"appium:deviceName": "iPhone Fast Simulator"
+"appium:platformVersion": "latest|current_major|previous_major"
+```
+
+For iPad:
+
+```java
+"appium:deviceName": "iPad Fast Simulator"
+"appium:platformVersion": "latest|current_major|previous_major"
+```
+
+This will assign a fast-booted Simulator that represents the latest/current major iOS version for the requested device by what both Apple has released and what Sauce Labs supports in our Simulator cloud. In the case of using previous_major platform version, it will represent the latest previous major iOS version for the requested device by what both Apple has released and what Sauce Labs supports in our Simulator cloud.
```

Comment:
```suggestion
This will assign a fast-booted Simulator that represents the `latest`/`current_major` iOS version for the requested device by what both Apple has released and what Sauce Labs supports in our Simulator cloud. In the case of using `previous_major` platform version, it will represent the latest previous major iOS version for the requested device by what both Apple has released and what Sauce Labs supports in our Simulator cloud.
```